app/serializers/olympian_stats_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class OlympianStatsSerializer
+
+  def self.serialize
+    output = { "olympian_stats": {}}
+    output[:olympian_stats][:total_competing_olympians] = Olympian.count
+    output[:olympian_stats][:average_weight] =
+      {
+        "unit": "kg",
```

Comment:
This line being hardcoded isn't awesome but strangely the dataset does not include an explicit weight unit, but Googled a couple of the athletes to check and these come across as kg.